sheet_to_triples/trans.py

```diff
@@ -162,3 +163,7 @@ def _iter_row_triples_some_subj(self, _f, params):
                 subj = _as_iri_or_none(_f, s, params)
                 if subj is not None:
                     yield (subj, _as_iri(_f, p, params), obj)
+
+    @property
+    def _skip_empty_rows(self):
```

Comment:
Can set a default value in `__init__` before the `setattr()` loop rather than needing to use another access interface?

Reply:
Yeah that's much more straightforward (and obvious), I'll blame `__slots__` being around and throwing me off.